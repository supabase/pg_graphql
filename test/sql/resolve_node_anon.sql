begin;

    create table account(
        id int primary key
    );


    insert into public.account(id)
    select * from generate_series(1,5);


    select gql.resolve($$
    {
      account(id: "WyJhY2NvdW50IiwgMV0=") {
        id
        nodeId
      }
    }
    $$);

rollback;
