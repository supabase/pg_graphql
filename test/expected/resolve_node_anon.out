begin;
    create table account(
        id int primary key
    );
    insert into public.account(id)
    select * from generate_series(1,5);
    select gql.resolve($$
    {
      account(id: "WyJhY2NvdW50IiwgMV0=") {
        id
        nodeId
      }
    }
    $$);
                                     resolve                                      
----------------------------------------------------------------------------------
 {"data": {"account": {"id": 1, "nodeId": "WyJhY2NvdW50IiwgMV0="}}, "errors": []}
(1 row)

rollback;
